codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketCloseFrameHandler.java

```diff
@@ -29,7 +29,9 @@
 
 /**
  * Send {@link CloseWebSocketFrame} message on channel close, if close frame was not sent before.
+ * @deprecated this class was merged into {@link WebSocketProtocolHandler} so should no longer be used
  */
+@Deprecated
 final class WebSocketCloseFrameHandler extends ChannelOutboundHandlerAdapter {
```

Comment:
@doom369 as this is package-private we can just remove the class imho.

Reply:
Removed